app/mailers/admin_mailer.rb

```diff
@@ -10,7 +10,11 @@ def notify_node_moderators(node)
     @node = node
     @user = node.author
     @footer = feature('email-footer')
-    moderators = User.where(role: %w(moderator admin)).collect(&:email)
+    all_moderators = User.where(role: %w(moderator admin)).collect(&:email)
+    moderators = []
+    all_moderators.each do |mod_user|
+      moderators << mod_user unless mod_user.has_tag('no-moderation-emails')
```

Comment:
then this to  `moderators << mod_user.email unless mod_user.has_tag('no-moderation-emails')`